DiaryAha/DiaryAha/Views/EditEntryView.swift

```diff
@@ -7,78 +7,112 @@
 
 import PhotosUI
 import SwiftUI
+import SwiftData
 
 struct EditEntryView: View {
-    @EnvironmentObject var entriesController: EntriesController
+    @Environment(\.modelContext) var modelContext
+
     @Binding var showNewEntry: Bool
     @State private var pickedPhotos = [PhotosPickerItem]()
-    @State private var entryPhotos = [Entry.PhotoMedia]()
+    @State private var entryPhotos = [Data]()
     @State private var entryTitle = ""
     @State private var entryBody = ""
     @State private var entryVibe: VibeImages = .happy
+    @State private var entryIdeas: [String] = []
 
     func onTapVibe(vibe: VibeImages) {
         entryVibe = vibe
     }
 
+    func getIdeas() {
```

Comment:
This looks familiar. :)

This a good example of why this code should be in an @ObservedObject instead of a view.

Reply:
I think this was a messup on my end not moving all the code. I meant to move this only to the GetIdeasView. Would I still need an observable?